scripts/integration_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftAWSLambdaRuntime open source project
+##
+## Copyright (c) 2017-2018 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+set +x -e
+
+for EXAMPLE in $(find Examples -type d -d 1);
+do
+	echo "Building $EXAMPLE"
+	pushd $EXAMPLE
+	LAMBDA_USE_LOCAL_DEPS=../.. swift build
+	popd
+done 
```

Comment:
what do you think about using a matrix to build them. this should make CI much faster. I don't wat to wait for the AWSSDK and Soto to succeed.

Reply:
are you talking about a CI Action (do you have an example?) or a shell script to run these in parallel ?

Reply:
Is this what you're referring too ?
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow